src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,55 @@
+package org.example.service;
+
+import org.example.model.Developer;
+
+import java.io.PrintWriter;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConnectionService {
+
+    private static final String DATABASE_NAME = "postgres";
+    private static final String DATABASE_USER = "postgres";
+    private static final String DATABASE_PASSWORD = "postgres";
+    private final Connection connection;
+
+    private static ConnectionService INSTANCE = null;
+
+    public ConnectionService() {
+        this.connection = connectToDb(DATABASE_NAME, DATABASE_USER, DATABASE_PASSWORD);
+    }
+
+    public Connection getConnection() {
+        return connection;
+    }
+
+    public Connection connectToDb(String dataaBaseName, String user, String pass) {
+        Connection connection = null;
+
+
+        try {
+
+            Class.forName("org.postgresql.Driver");
+            connection = DriverManager.getConnection("jdbc:postgresql://developers-db:5432/" + dataaBaseName, user, pass);
```

Comment:
Extract the creation of the DB URL into a separate method

Reply:
Please check it

Reply:
fix